README.md

```diff
@@ -54,9 +54,10 @@
 
 # LTS Plan
 
-New semver-major releases of Node.js are cut from `master` every six months.
-New even-numbered versions (e.g. v6, v8, v10, etc) are cut in April. New 
-odd-numbered versions (e.g. v5, v7, v9) are cut in October.
+New semver-major releases of Node.js are cut from `master` every six months
+and become the "current" release.  New even-numbered versions (e.g.  v6, v8,
+v10, etc) are cut in April.  New odd-numbered versions (e.g.  v5, v7, v9)
+are cut in October.
```

Comment:
Any info on the Current branch should be in the main repo IMO.

Anyone can propose minor or patch releases on it, although that is not clear. If no-one else does, one of the releasers will. We aim for every week to every other week, although sometimes releases slip to 3.

Anything that can land and isn't breaking (or depending on a breaking change), we land it on Current. (Occasionally we decide to do a patch anyways rather than a minor if there are few minors, releaser's discretion.)


> and for contributors or observers to be able to predict when commits from any particular PR (barring negative reports on it after merging to master) will hit any particular release stream would be really helpful.

If you want something expedited, just propose a release... or even do some of the pre-release work. :P